checkers/mozlenium/test.js

```diff
@@ -0,0 +1,18 @@
+// EXAMPLE copied script
```

Comment:
I'm ok with including examples but this should be clearly marked as doc, lets either make an examples directory in /checkers/mozlenium, or move this here, and replace demo-check.js: https://github.com/mozilla-it/mozalert/tree/main/examples

Reply:
I'm confused about what you mean here. 
- Make an examples directory. Check.
- remove "check.js" in favour of "demo-check.js"
- the rest is confusing to me